styx-common/src/main/java/com/spotify/styx/storage/Storage.java

```diff
@@ -116,14 +117,14 @@
    * @return true if the queried workflow is enabled
    */
   @Deprecated
-  boolean enabled(WorkflowId workflowId) throws IOException;
+  boolean isWorkflowEnabled(WorkflowId workflowId) throws IOException;
```

Comment:
Ah, forgot about the strings in `MeteredStorage`. Will have another pass at this.